components/brave_wallet/rust/BUILD.gn

```diff
@@ -13,6 +13,7 @@ rust_static_library("rust_lib") {
   cxx_bindings = [ "lib.rs" ]
```

Comment:
in a follow-up `rust` does not belong at the top level of the component like this. Also it doesn't make sense for code outside of rust/third_party to have Cargo.toml anymore. I guess I missed this when I converted everything over to the upstream build, but this should go in brave_wallet/browser/rust

Reply:
> Also it doesn't make sense for code outside of rust/third_party to have Cargo.toml anymore.

FWIW I disagree. I often use `Cargo.toml` under the components tree for local testing and review. It's faster to iterate that way and different tooling is available. We do need a lint at least if we're going to maintain both builds, of course, or things will get out of sync as you've noted.

Reply:
Things have changed a bit in regards to Cargo.toml so this is ok now with recent chromium updates